Makefile

```
@@ -8,3 +8,7 @@ godoc:
 	go install golang.org/x/tools/cmd/godoc@latest
 	# http://localhost:6060/pkg/github.com/smartcontractkit/chainlink-relay/
 	godoc -http=:6060
+
+.PHONY: mockery
+mockery: $(mockery) ## Install mockery.
+	go install github.com/vektra/mockery/v2@v2.28.1
```

New patch version released a few days ago:
```suggestion
	go install github.com/vektra/mockery/v2@v2.28.2
```

---

Thanks Jordan for the heads up! will update that